### DISCUSSION TITLE
systemd 非 root 用户最小权限运行所需的权限

### DISCUSSION THREAD

**Discussion by Rirmach** [pymumu/smartdns#1703] [Q&A] [answered]
除保证缓存、日志文件读写权限外，设置了 CAP_NET_ADMIN, CAP_NET_BIND_SERVICE, CAP_NET_RAW 三项，是否满足 smartdns 的运行？

**Comment by pymumu** [comment, marked as answer]
smartdns配置文件设置user nobody

**Reply by Rirmach** [reply]
不过最好还是建议后续在 systemd 配置中避免使用 nobody，目前 systemd 已经报 `Special user nobody configured, this is not safe`

**Reply by pymumu** [reply]
不是在systemd中设置nobody
是在/etc/smartdns/smartdns.conf中设置
